js/components/MilkdropWindow/index.js

```diff
@@ -27,7 +28,14 @@ class MilkdropWindow extends React.Component {
         );
         this.visualizer.connectAudio(analyserNode);
         this.visualizer.loadPreset(reactionDiffusion2, 0);
-        this._renderViz();
+        // Kick off the animation loop
+        const loop = () => {
+          if (this.props.status === "PLAYING") {
```

Comment:
In Winamp, if you press stop, does it clear the Milkdrop window, or leave it with whatever it was showing last?

Reply:
I'll need to double check later tonight, I believe it just keeps whats on the screen and then picks up when the audio restarts.

I'll check tonight and confirm